appservice/src/createAppService/AppServicePlanSkuStep.ts

```diff
@@ -21,6 +22,11 @@ export class AppServicePlanSkuStep extends AzureWizardPromptStep<IAppServiceWiza
                 };
             });
 
+            if (wizardContext.newSiteOS === WebsiteOS.linux) {
+                // Free tier is not supported for Linux asp's
+                pricingTiers.shift();
```

Comment:
I'm not sure I'm the biggest fan of just 'shift'. It seems prone to bugs. For example - what if we re-order this again, or what if there are multiple free tiers at some point? I'd prefer to explicitly filter out all items that have 'free' as the tier